lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -204,6 +169,7 @@ class FirstMeaningfulPaint extends Audit {
     return {
       navigationStart,
       firstContentfulPaint,
+      firstMeaningfulPaint,
       layouts,
```

Comment:
layouts and paints arent used now
and more notably i think you killed off the entire `findFirstMeaningfulPaint` function.

probably a few things after that, too. 

wanna nuke all that?